hybridse/src/codegen/aggregate_ir_builder.cc

```diff
@@ -236,34 +232,25 @@ class StatisticalAggGenerator {
 
     void GenInitState(::llvm::IRBuilder<>* builder) {
         for (size_t i = 0; i < col_num_; ++i) {
+            // any of min/max/avg/sum/count need the count info
+            count_state_[i] = GenCountInitState(builder);
+
             if (!sum_idxs_[i].empty()) {
                 sum_states_[i] = GenSumInitState(builder);
             }
             if (!avg_idxs_[i].empty()) {
                 if (col_type_ == ::hybridse::node::kDouble) {
```

Comment:
why only judge `kDouble` here.

Reply:
the idea is generally:
- if column type is double, `avg` and `sum` can share the sum value

on output, avg requires `sum` and `count` to be float